src/components/Pages/Unit/widgets/Recommended/Main/DisplayRecommended.js

```diff
@@ -123,19 +123,27 @@ const RecommendedPlaylist = (recommendForUnit, units, selected, t, chronicles, v
                     asList
                     label={unitLabels[index]}
                     size={'small'} />
-                  : (IsUnitContentType(unit.content_type) ?
+                  : (unit.content_type && IsUnitContentType(unit.content_type) ?
```

Comment:
you don't want add this check on IsUnitContentType?

Reply:
No, as here we check simple string which the the content_type string. unit may be different things ,e.g., lesson, topic, source ....